### DISCUSSION TITLE
Pricing en Fixed Footer Layout

### DISCUSSION THREAD

**Discussion by JoseGonzalz** [Telefonica/mistica-design#1648] [Proposals]
### Goal

Mejorar la jerarquía de la info del pricing que aparece en este componente en conjunto con las pantallas donde se usa.

### Context

Es un componente que se utiliza en muchos flujos que tienen que ver con la contratación de servicios, donde incluso a veces es el importe total correspondiente a la suma de varios elementos como facturas.

En pantallas anteriores vemos que este concepto de pricing o importe está mucho más presente visualmente y tiene una posición más importante en la jerarquía de información, sin embargo en el componente actual, en conjunto con el resto de elementos, queda poco destacado.

### Impact

Crear un footer donde se refleje información importante como el pricing o cantidad total de una manera más visible y diferenciada de importes secundarios (como importe de una factura por ejemplo)

### Discussion Questions

Podríamos modificar el estilo actual text-preset-3-medium (que es el mismo que se usa en row list y en boxed row para mostrar el precio de un servicio o factura único) por un text-preset-5 para crear una mejor jerarquía en el Footer Fixed Layout.

![asisTobe](https://github.com/Telefonica/mistica-design/assets/129079621/fa7ffe14-3519-481e-9897-3405081cf70f)

### Do you have a visual proposal?

https://www.figma.com/file/cL8KKUna805Zf9ezYjML1r/Mistica-Requests?type=design&node-id=1-464&mode=design

**Comment by yceballost** [comment]
te vas a reir, pero la realidad es que el fixedFooterLayout como tal no define ese diseño del precio. Es un ejemplo que se quedó en su día en la librería de Figma y sigue ahí. Es decir, el fixedFooterLayout como su nombre indica es un layout y nada más, que lo que hace es posicionarse abajo de la pantalla con una sombrita cuando hay contenido en el scroll, ese es el componente únicamente. Lo que se meta dentro es cosa de cada producto.

Dicho esto, veo dos caminos para solucionar esto.
1. Eliminar de la librería ese componente con el diseño del price y dejar únicamente la variante del slot (que al final es lo que es)
2. Podemos cambiar sin problema el diseño del price en ese ejemplo que hay la librería de Figma pero al final estaríamos alargando futuros cambios que a Mística no deberían llegar

Yo creo que esa variante no debería de estar en la librería y que cada producto se lo construya como necesite, en vuestro caso, esa construcción debería de ir en la novum extended library (aunque en desarrollo existe la librería de global checkout y en diseño no existe, pero eso es algo que podemos hablar en otro momento jaja)

**Reply by JoseGonzalz** [reply]
Pues... lo que decidáis bien estará :)

**Reply by AnaMontes11** [reply]
@JoseTelefonica vamos a ampliar la documentación en Brand Factory para que quede claro qué es el fixedFooterLayout y se trate unicamente como un slot donde cada producto pueda meter lo que quiera. Además vamos a deprecar las variantes de Figma para que no den lugar a confusión.
Como solución a tu necesidad puedes crearte tu propio componente con la propuesta que nos comentas y reemplazarlo dentro del slot del fixedFooterLayout.

**Reply by JoseGonzalz** [reply]
Genial!! Tnks!
